www/adminops

```diff
@@ -1849,7 +1849,7 @@ else if (isset($_REQUEST['users'])) {
                filterID, filterName, ckBoxName, showName, endDate,
                filterType, explanation
              from filters
-             where filterid = '$qFilterID'");
+             where filterid = '$qFilterID'", $db);

```

Comment:
I was unable to test this feature locally. It looks like it ought to work if you go to http://localhost:8080/adminops and click on "View/edit game filter list" http://localhost:8080/adminops?filters but if you go there, it says, "Table 'ifdb.filters' doesn't exist" 

And, indeed, that's not a table, and never has been since our first data drop from MJR in January, so I think this is just dead code.

Reply:
Should I delete everything related to that query in a separate commit?

Reply:
Eh, it's certainly not the only dead code in the code base. If you went on a dead-code hunt as a separate PR, that would be welcome